ISSUE_TEMPLATE/stable.md

```diff
@@ -0,0 +1,97 @@
+First, verify that you meet all the [prerequisites](https://github.com/coreos/fedora-coreos-streams/blob/master/release-prereqs.md)
+
+Name this issue `stable: new release on YYYY-MM-DD` with today's date. Once the pipeline spits out the new version ID, you can append it to the title e.g. ` (31.20191117.3.0)`.
+
+# Pre-release
+
+## Promote testing changes
+
+From the checkout for `fedora-coreos-config` (replace `upstream` below with
+whichever remote name tracks `coreos/`):
+
+- [ ] `git fetch upstream`
+- [ ] `git checkout stable`
+- [ ] `git reset --hard upstream/stable`
+- [ ] `/path/to/fedora-coreos-releng-automation/scripts/promote-config.sh testing`
+- [ ] Sanity check promotion with `git show`
+- [ ] Open PR against the `stable` branch on https://github.com/coreos/fedora-coreos-config
+- [ ] Post a link to the PR as a comment to this issue
+- [ ] Ideally have at least one other person check it and approve
+- [ ] Once CI has passed, merge it
+
+## Build
+
+- [ ] Start a [pipeline build](https://jenkins-fedora-coreos.apps.ci.centos.org/job/fedora-coreos/job/fedora-coreos-fedora-coreos-pipeline/build?delay=0sec) (select `stable`, leave all other defaults)
+- [ ] Post a link to the job as a comment to this issue
+- [ ] Wait for the job to finish
+
+## Sanity-check the build
+
+Using the [the build browser](https://builds.coreos.fedoraproject.org/browser) for the `stable` stream:
+
+- [ ] Verify that the parent commit and version match the previous `stable` release (in the future, we'll want to integrate this check in the release job)
+- [ ] Check [kola AWS run](https://jenkins-fedora-coreos.apps.ci.centos.org/job/fedora-coreos/job/fedora-coreos-fedora-coreos-pipeline-kola-aws) to make sure it didn't fail
+
+# ⚠️ Release ⚠️
+
+IMPORTANT: this is the point of no return here. Once the OSTree commit is
+imported into the unified repo, any machine that manually runs `rpm-ostree
+upgrade` will have the new update.
+
+## Importing OSTree commit
+
+In the future, the OSTree commit import will be integrated in the release job.
+
+- [ ] Open an issue on https://pagure.io/releng to ask for the OSTree commit to be imported (include a URL to the .sig which should be alongside the tarfile in the bucket and signed by the primary Fedora key, also include the ref you'd like to import, i.e. `fedora/x86_64/coreos/stable`)
```

Comment:
Doubt: how will this work once we have multiple architectures?

Reply:
By then, we should have the [OSTree importer](https://github.com/coreos/fedora-coreos-tracker/issues/199) in place, which will take care of importing refs for all the arches. (Obviously, it totally sidesteps the question of how to create those refs in the first place; that's https://github.com/coreos/fedora-coreos-tracker/issues/262)